spec/services/permissions_checker_spec.rb

```diff
@@ -75,7 +80,8 @@
           permission_names: [],
           user_id: super_admin_user.id,
           friendly_id: '',
-          record_id: ''
+          record_id: '',
+          current_provider: super_admin_user.provider
```

Comment:
Maybe we can add another check to ensure that we will catch users trying to access resources on a different provider,